integrations/elasticsearch/src/haystack_integrations/components/retrievers/elasticsearch/bm25_retriever.py

```diff
@@ -60,6 +63,7 @@ def __init__(
             for more details.
         :param top_k: Maximum number of Documents to return.
         :param scale_score: If `True` scales the Document`s scores between 0 and 1.
+        :param filter_policy: Policy to determine how filters are applied.
```

Comment:
Would it help to give possible values for this, like REPLACE or MERGE.

Reply:
That's a good idea but it gets outdated fast because `FilterPolicy` changes and we have to do more maintenance work. As it is defined type users can simply look up `FilterPolicy` enum and see values available.

Reply:
Makes sense.